src/util.cc

```diff
@@ -475,4 +480,19 @@ void SetConstructorFunction(Local<v8::Context> context,
   that->Set(context, name, tmpl->GetFunction(context).ToLocalChecked()).Check();
 }
 
+const char* FindSingleExecutableCode(size_t* size) {
+  if (single_executable_application_code_loaded == false) {
```

Comment:
I think we need the concept of a "fuse" here. Something like what I had in src/node_single_executable_application.cc on line 43, as borrowed from electron. 

That ensures that if you are not using SEA, the overhead is as close to possible as 0.  It does mean there is an extra step in the bundling but I don't see that as a big issue.

Reply:
@mhdawson does https://github.com/nodejs/postject/pull/59 look good to you?